main.go

```diff
@@ -210,16 +221,47 @@ func main() {
 			Port:    webhookPort,
 			TLSOpts: tlsOptionOverrides,
 		}),
-		LeaderElection:          enableLeaderElection,
-		LeaderElectionID:        leaderElectionID,
-		LeaderElectionNamespace: leaderElectionNamespace,
-		HealthProbeBindAddress:  healthAddr,
+		LeaderElection:                enableLeaderElection,
+		LeaderElectionID:              leaderElectionID,
+		LeaderElectionNamespace:       leaderElectionNamespace,
+		LeaderElectionReleaseOnCancel: leaderElectionReleaseOnCancel,
```

Comment:
TBH I'm struggling to think of a reason we couldn't set this to true unconditionally.

The [docs](https://github.com/kubernetes-sigs/controller-runtime/blob/479b723944e34ae42c9911fe01228ff34eb5ca81/pkg/manager/manager.go#L197-L201) say:

> This requires the binary to immediately end when the Manager is stopped, otherwise this setting is unsafe.

As far as I can tell we never stop the manager and leave other stuff running, so there should be no downside to always setting this.

Even the terminology here ("on cancel") is referring to internal golang stuff that doesn't really make sense as a CLI flag.

Reply:
Done